zone/client_packet.cpp

```diff
@@ -899,6 +930,9 @@ void Client::CompleteConnect()
 
 	entity_list.ScanCloseMobs(close_mobs, this, true);
 
+	entity_list.SendToGuildTitleDisplay(this);
+	SendGuildSpawnAppearance();
```

Comment:
Thanks.  I have been thinking about this and think it wise to update the zone spawn packets and spawn struct as it may resolve the guild window as well.  That said, I can spend some time taking a look.  I would like to dig deeper on the spawn functionality.  Thanks hg.

Reply:
Updated to use the zone spawn for guild display and removed unused functions.  Thanks for the advice hg as this will also benefit my trader/buyer/bazaar work.

Reply:
Looks like the wrong line was removed here:
`entity_list.ScanCloseMobs(close_mobs, this, true);`
should be added back
`SendGuildSpawnAppearance();`
should be removed after the zone spawn changes